android/src/main/kotlin/im/nfc/flutter_nfc_kit/FlutterNfcKitPlugin.kt

```diff
@@ -36,6 +37,8 @@ class FlutterNfcKitPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
         private var pollingTimeoutTask: TimerTask? = null
         private var tagTechnology: TagTechnology? = null
         private var ndefTechnology: Ndef? = null
+        private var isMifareClassic: Boolean = false;
```

Comment:
Since mifare classic and ultralight are orthogonal, and considering we plan to add ntag support, would it be better to use a enum value here? @Harry-Chen